src/net_stat.cc

```diff
@@ -128,6 +136,22 @@ void parse_net_stat_arg(struct text_object *obj, const char *arg,
     arg = e_iface;
   }
 
+  if (0 == strncmp(arg, "${iface", 7)) {
+    for (; *arg_ptr; arg_ptr++) {
```

Comment:
@su8 I have to go afk for an hour or two. Moving engine. :sweat_smile: Bbl.

Reply:
@su8 I just tried this. It does not work for me because `iface` PR were merged immediately with bad commits. 
```
conky.config = {out_to_x=false,out_to_console=true}
conky.text = [[
    ${iface}
    ${iface 0}: ${upspeedf ${iface 0}} // ${downspeedf ${iface 0}
    ${iface 1}: ${upspeedf ${iface 1}} // ${downspeedf ${iface 1}
    ----------
]]
```
```
$ conky -c ~/conky/iface-downspeedf.conf
    eno1
    : 0.0 // 0.0
    eno1
    : 0.0 // 0.0
    eno1
    : 0.0 // 0.0
^C    eno1
```

Reply:
@su8 There is something wrong with the example above. I have `test` at end of `conky.text` and it does not show up in the output.

Reply:
```bash

    eth0
    : 0.0 // 0.0
    eth0
    : 4.2 // 0.0
    eth0
    : 4.4 // 0.0
    eth0
    : 1.3 // 0.0
    eth0
    : 5.7 // 0.0
    eth0
    : 7.6 // 0.0

```

Reply:
Bbl now! It's working now. Weird.

Reply:
Close this commit then ?

Reply:
Single `${iface 1}: ${upspeedf ${iface 1}} // ${downspeedf ${iface 1}` works fine, but not when using it multiple times.

Reply:
Same bug as the yesterdays one :sob:

Reply:
>Bbl now! It's working now. Weird.

Only the upspeed is working, the downspeed always shows 0.

Reply:
Back. The trip was a waste. We need more people to haul the engine out of the basement. Trying again this evening presumably with more people.

This works fine now, but I had it not working in the past. Maybe because I was using `master` or had too many interfaces. I kept testing this and it seems to be okay.